portal/portal-impl/impl/src/bundle/profile-popup.properties

```diff
@@ -2,18 +2,18 @@ connect = Connect
 connect_tooltip = Click to send a connection request to this person
 remove = Remove Connection
 remove_tooltip = Click to remove this person as a connection
-accept = Accept
+accept = Accept Connection
 accept_tooltip = Click to accept this person's connection request
-ignore = Ignore
+ignore = Ignore Connection
 ignore_tooltip = Click to ignore the connection request from this person
-cancel = Cancel
+cancel = Cancel Connection
 cancel_tooltip = Click to cancel your connection request to this person
 
 profile = Profile
 profile_tooltip = Click on this to view this person's Profile
 email = Email
 email_tooltip = Click on this to send this person an email
 facebook_tooltip = Click on this to launch this person's Facebook page
-no_facebook_tooltip = This person has not setup a Facebook link
-twitter_tooltip = Click on thie to launch this person's Twitter feed
-no_twitter_tooltip = This person has not setup a Twitter link
+no_facebook_tooltip = There is no Facebook account on this profile
```

Comment:
How about 'attached to this profile'? The aim's to just clue someone in where to go to add it.